MonoGame.Framework.Content.Pipeline/Utilities/BcnHelpers.cs

```diff
@@ -0,0 +1,61 @@
+// MonoGame - Copyright (C) MonoGame Foundation, Inc
+// This file is subject to the terms and conditions defined in
+// file 'LICENSE.txt', which is part of this source code package.
+
+using System;
+using System.IO;
+using BCnEncoder.Encoder;
+using BCnEncoder.Shared;
+using KtxSharp;
+using Microsoft.Xna.Framework.Content.Pipeline.Graphics;
+
+namespace Microsoft.Xna.Framework.Content.Pipeline.Utilities;
+
+/// <summary>
+/// The <see cref="Encode"/> method uses the https://github.com/Nominom/BCnEncoder.NET/tree/master
+/// library to encode various block based compression formats
+/// </summary>
+public static class BcnUtil
```

Comment:
Should this be internal and not public?

Do we want to make this part of the public API?

@cdhanna @harry-cpp @mrhelmut ?

Reply:
I think it should be internal , that is the same for the other helpers which have been added.